<launch>
  <node
     ns="/wide"
     pkg="stereo_image_proc"
     type="stereo_image_proc"
     name="stereo_image_proc_wide"
     respawn="true" >
    <param name="approximate_sync" value="true" /> 
  </node>

  <node
     ns="/narrow"
     pkg="stereo_image_proc"
     type="stereo_image_proc"
     name="stereo_image_proc_narrow"
     respawn="true" >
    <param name="approximate_sync" value="true" /> 
</node>



  <!-- optional visualization tools -->

  <group if="$(optenv ENABLE_CONSOLE 0)">
    <node
       pkg="rxtools"
       type="rxconsole"
       name="rxconsole" />
  </group>

</launch>
